src/Tabs/Tabs.js

```diff
@@ -33,7 +34,15 @@ export class Tabs extends Component {
           {contents[this.state.id]}
         </div>
       </section>
-
     );
   }
 }
+
+Tabs.propTypes = {
+  children: PropTypes.element.isRequired,
```

Comment:
you say to react that children is only one element, but put inside tabs more than one element